src/cmds/open.ts

```diff
@@ -21,21 +23,36 @@ export default class OpenCommand extends Command {
     this.cmdCategory = CommandCategories.UTILITIES;
     this.position = 1;
 
-    this.args = [];
+    this.args = [
+      {
+        name: 'dash',
+        type: Boolean,
+        description: 'Opens the current project dashboard.',
```

Comment:
Small copy suggestion:

```suggestion
        description: 'Opens your current ReadMe project dashboard.',
```